src/content/en/2020/accessibility.md

```diff
@@ -8,17 +8,442 @@ authors: [oluoluoxenfree, alextait1]
+
+ARIA attributes can be assigned to HTML elements to enhance the accessibility of the interface. Respecting the first rule of ARIA, they should not be used to achieve something that can be done with native HTML.
+
+##### Labeling and describing elements with ARIA
+
+The browser’s accessibility tree has a computation system that assigns the accessible name (if there is one) to a control, widget, group, or landmark such that it can be announced by assistive technology. There is a specificity ranking that happens to determine which value is assigned to the accessible name. 
+
+The accessible name can be derived from an element’s content (such as button text), an attribute (such as an image alt text value), or an associated element (such as a programmatically associated label for a form control. For more information about accessible names see Léonie Watson’s article, [What is an accessible name?](https://developer.paciellogroup.com/blog/2017/04/what-is-an-accessible-name/)
+
+We can also use ARIA to provide accessible names for elements. There are 2 ARIA attributes that accomplish this, [aria-label](https://www.w3.org/WAI/GL/wiki/Using_aria-label_to_provide_labels_for_objects), [aria-labelledby](https://www.w3.org/WAI/GL/wiki/Using_aria-labelledby_to_provide_a_name_for_user_interface_controls) . Either of these attributes will “win” the accessible name computation and override the natively derived accessible name, so use them with caution and be sure to test with a screen reader or look at the accessibility tree to confirm that the accessible name is what was expected. When using ARIA to name an element, it is important to ensure that the [WCAG 2.5.3, Label in Name](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html) criterion has not been violated, which expects visible labels to be at least a part of its accessible name. 
+
+
+The `aria-label` element allows a developer to provide a string value and this will be used for the accessible name for the element. We found that 40.44% of desktop pages and 38.72% of mobile home pages had at least one element with the aria-label attribute, making it the most popular ARIA attribute for providing accessible names. 
+
+The `aria-labelledby` attribute accepts an id reference as its value which associates it with another element in the interface to provide it’s accessible name. The element becomes “labelled by” this other element which supplies its accessible name. We found that 17.73% of desktop pages and 16.21% of mobile pages had at least one element with the `aria-labelledby` attribute. 
+
+Again the first rule of ARIA should be respected. If the element can derive it’s accessible name without needing ARIA, this is preferable. For example a `<button>` which is not a graphical element should get its accessible name from it’s text content rather than an ARIA attribute. Form elements should derive their accessible names from properly associated `<label>` elements whenever possible. 
```

Comment:
```suggestion
Again, the first rule of ARIA should be respected. If the element can derive its accessible name without needing ARIA, this is preferable. For example a `<button>`, which is not a graphical element, should get its accessible name from its text content rather than an ARIA attribute. Form elements should derive their accessible names from properly associated `<label>` elements whenever possible. 
```